### PR TITLE
third party api

### PR DESCRIPTION
**Pull request by robbinlol**
using python with third party api to realize wrapper

### CODE REVIEW

**Comment by whybeyoung** [comment]
please config you git user. requires all commit messages to contain the Signed-off-by line with an email address that matches the commit author.

**Comment by robbinlol** [comment]
add the config description.

**Comment by robbinlol** [comment]
add the readme file.

File: demo/music_api/aiges.toml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+numaNode = 0
+realTimeRlt = false
+usrCfg = ""
+sub = "common"
+
+[wrapper] #wrapper示例配置
+"common.lic" = "10"
+"resource./msp/resource/sms/acmod_16KPowerFlt_sms_RNN.bin" = "HMM_16K"
+"resource./msp/resource/sms/wfst.bin" = "WFST"
+<<<<<<< HEAD
```
Comment (whybeyoung) [inline demo/music_api/aiges.toml]: resolve it.

File: demo/music_api/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+### 利用第三方API实现Python加载器插件
+- 三方API的文档可以[参考](https://docs.acrcloud.cn/api-reference/identification-api/)
```
Comment (whybeyoung) [inline demo/music_api/README.md]: 这里可以稍微加一些介绍性语言，介绍 这个加载器 加载三方api的目的是啥？为啥要用这个加载器，场景介绍。
不然用户会很模糊，不明所以

**Review by whybeyoung — CHANGES_REQUESTED** [review]

**Comment by robbinlol** [comment]
解决了冲突，完善了README

**Pull request merged**